package.json

```diff
@@ -2,14 +2,14 @@
 	"name": "vscode-elrond-ide",
 	"displayName": "MultiversX IDE",
 	"description": "MultiversX IDE for developing Smart Contracts",
-	"version": "0.18.0",
+	"version": "0.19.0",
 	"publisher": "Elrond",
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/multiversx/mx-ide-vscode.git"
 	},
 	"engines": {
-		"vscode": "^1.64.0"
+		"vscode": "^1.76.2"
```

Comment:
Poate ar fi bine sa folosesti versiune fixa aici ?

Reply:
Here, it's the only place where we use a non-fixed version (so that the extension can be installed on newer versions of VSCode - at least, from my understanding).